<launch>
 <param name = "product" type = "string" value = "LD00" />
 <node name="LD00" pkg="ldlidar" type="ldlidar" output="screen">
 </node>
 
</launch>
